controllers/configurationpolicy_controller.go

```diff
@@ -1106,6 +1106,14 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 		numCompliant := 0
 		numNonCompliant := 0
 		handled := false
+		// map raw object to a resource, generate a violation if resource cannot be found
+		mapping, statusUpdateNeeded := r.getMapping(objectT.ObjectDefinition, &plc, indx)
+
+		unstruct, err := unmarshalFromJSON(objectT.ObjectDefinition.Raw)
+		if err != nil {
+			os.Exit(1)
```

Comment:
I know you didn't write this code but could you please change this to a panic instead?